app/tests/unit/containers/User/UserProfileDropdown.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {UserProfileDropdownComponent} from 'containers/User/UserProfileDropdown';
+import {UserProfileDropdown_user} from '__generated__/UserProfileDropdown_user.graphql';
+
+const user = {
+  firstName: 'Test',
+  lastName: 'Tester',
+  emailAddress: 'really-long-email-to-see-wrapping@button.is',
+  ' $refType': 'UserProfileDropdown_user'
+};
+let render;
+describe('UserProfileDropdown', () => {
+  it('matches mobile snapshot', () => {
+    render = shallow(
+      <UserProfileDropdownComponent user={user as UserProfileDropdown_user} />
+    );
+    expect(render).toMatchSnapshot();
+  });
+
+  it('matches desktop snapshot', () => {
+    render = shallow(
+      <UserProfileDropdownComponent user={user as UserProfileDropdown_user} />
+    );
+    expect(render).toMatchSnapshot();
+  });
```

Comment:
I don't see the difference between those two tests. If you're looking for visual differences, they will be in the happo screenshots

Reply:
yeah forgot I had this in there, was harder to change the jsdom window size than I thought. A better test for cypress I guess.